lib/parseTriggers.js

```diff
@@ -4,15 +4,18 @@ var FirebaseError = require('./error');
 var fork = require('child_process').fork;
 var path = require('path');
 var RSVP = require('rsvp');
+var _ = require('lodash');
 
 var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
 
 module.exports = function(projectId, sourceDir, configValues, firebaseConfig) {
   return new RSVP.Promise(function(resolve, reject) {
     var env = {
-      CLOUD_RUNTIME_CONFIG: JSON.stringify(configValues),
```

Comment:
Since you’re removing this one, it may be better to fit this object declaration in one line.